packages/atomic/src/index.html

```diff
@@ -106,6 +107,68 @@
         margin: 10px 0;
       }
 
+      @media only screen and (max-width: 1024px) {
+        atomic-search-interface {
+          grid-template-rows: 16px 50px 16px auto 1fr;
+          grid-template-columns: minmax(0, 100%);
+        }
+
+        atomic-facet-manager,
+        .header,
+        atomic-search-box,
+        .results-container {
+          grid-column: 1 / auto;
+        }
+
+        atomic-breadcrumb-manager,
+        atomic-query-summary,
+        atomic-no-results,
+        atomic-did-you-mean,
+        atomic-query-error,
+        atomic-sort-dropdown,
+        atomic-facet-manager {
+          padding: 0 15px;
+        }
```

Comment:
Should the padding be inside the web-components so that they are padded by default?

Reply:
I don't think so/all components should take the "maximum size they are allowed to", as a rule of thumb, and should not assume that "adding a padding of 15 will make it look good", because it might not be true in some cases.


But, this is all problems we'll have to solve/discuss when we work on v1 and determine how we want to do out different layouts.

Reply:
I think we're talking about different things though: `width` 100% vs. `padding`. I am assuming this padding is applied inside the facet? Or am I off by 1 and it is outside it, affecting the shadow root vs. the root element inside the shadow dom?

All this to say, I think components should look good by default, not require additional base styles. e.g. I would consider a facet that has padding on larger screens, and then loses it on smaller screen part of a well-designed component lib.

Reply:
> All this to say, I think components should look good by default, not require additional base styles. e.g. I would consider a facet that has padding on larger screens, and then loses it on smaller screen part of a well-designed component lib.

Here, the padding is added to add "white space" between it and other elements in the page, which depends on how the layout is done.

We could have used margin.

How much whitespace there should be between elements/between elements and the border of the page etc, it's all layout dependant.

In this case, for example:

By default the facets (facet manager) takes the full width, has no margin, has no padding.
If you put it inside the <body> of the page, it's going to go all the way to the very border of the body/page.

For this layout, we arbitrarily decide that we want to add a whitespace of 15px horizontally so that it's nicer to look at.

Now, lets assume the same facets were added in an arbitrary div called `.facets-container`, which itself has a padding of 15px.

If the facet manager also had, internally, a padding of 15px, you would effectively end up with 30px of "white space"

Reply:
ok, the confusion is stemming from `padding` (aka inside the borders of the component) vs. `margin` (outside the component). Anything related to layout should be outside of the component, I 100% agree.